python/_tskitmodule.c

```diff
@@ -8724,6 +8725,59 @@ TreeSequence_get_individuals_time(TreeSequence *self)
     return ret;
 }
 
+static PyObject *
+TreeSequence_get_individuals_nodes(TreeSequence *self)
+{
+    PyObject *ret = NULL;
+    PyArrayObject *ret_array = NULL;
+    npy_intp dims[2];
+    tsk_size_t ploidy;
+    tsk_size_t max_ploidy = 0;
+    tsk_id_t *node_mem;
+    tsk_size_t j;
+    const tsk_size_t num_individuals
```

Comment:
Here's your problem - you're calling tree sequence methods before ``TreeSequence_check_state``

Reply:
*facepalm* Thanks.